server/src/com/thoughtworks/go/server/websocket/ConsoleLogSender.java

```diff
@@ -113,11 +113,7 @@ public void process(final SocketEndpoint webSocket, JobIdentifier jobIdentifier,
     }
 
     private boolean doesLogExists(JobIdentifier jobIdentifier) {
-        try {
-            return consoleService.consoleLogFile(jobIdentifier).exists();
-        } catch (IllegalArtifactLocationException e) {
-            return false;
-        }
+        return consoleService.doesLogExists(jobIdentifier);
```

Comment:
Replace all occurrences of `consoleService.consoleLogFile(jobIdentifier).exists()` with something like above.